apps/api-extractor/src/api/model/ApiDocumentedItem.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as tsdoc from '@microsoft/tsdoc';
+import { ApiItem, IApiItemOptions, IApiItemJson } from './ApiItem';
+import { AedocDefinitions } from '../../aedoc/AedocDefinitions';
+
+/** @public */
+export interface IApiDocumentedItemOptions extends IApiItemOptions {
+  docComment: tsdoc.DocComment | undefined;
+}
+
+export interface IApiDocumentedItemJson extends IApiItemJson {
+  docComment: string;
+}
+
+/** @public */
+export class ApiDocumentedItem extends ApiItem {
+  private _tsdocComment: tsdoc.DocComment | undefined;
+
+  /** @override */
+  public static onDeserializeInto(options: Partial<IApiDocumentedItemOptions>,
+    jsonObject: IApiItemJson): void {
+
+    ApiItem.onDeserializeInto(options, jsonObject);
+
+    const documentedJson: IApiDocumentedItemJson = jsonObject as IApiDocumentedItemJson;
+
+    if (documentedJson.docComment) {
+      const tsdocParser: tsdoc.TSDocParser = new tsdoc.TSDocParser(AedocDefinitions.tsdocConfiguration);
+      const parserContext: tsdoc.ParserContext = tsdocParser.parseString(documentedJson.docComment);
+
+      // TODO: Warn about parser errors
+
```

Comment:
I'm planning to do it after merging this PR, but before the non-beta release.  It's not difficult, but it's something I want to code review in isolation, because the AE6 design isn't great and possibly will have changes.

---
In reply to: [236468054](https://github.com/Microsoft/web-build-tools/pull/952#discussion_r236468054) [](ancestors = 236468054)